svthalia/concrexit
# Keep HTML tags in events no_registration_message

Closes #2274 in literally one line



### Summary
Do not HTML escape no_registration_message

### How to test
Steps to test the changes you made:
1. Create an event with HTML in the no_registration_message
2. Go to its page
3. Observe that the HTML is still there
